docs/0-what-is/1-getting-started.mdx

```diff
@@ -5,81 +5,183 @@ sidebar_position: 0
+          # Optional
+          format.type: 'structured'
+          # The maximum rate in records per second, at which records are
+          # generated (0 means no rate limit).
+          # Type: float
+          # Optional
+          rate: '1'
+      - id: example-destination
         type: destination
-        # use the built-in file plugin as the destination
-        plugin: builtin:file
-        settings:
-          # write data to example.out
-          path: ./example.out
-          # output the raw payload as a string
-          sdk.record.format: template
-          sdk.record.format.options: '{{ printf "%s" .Payload.After }}'
+        plugin: "log"
 ```
 
-Now start Conduit. You should see a log line saying that the pipeline
-`file-to-file` was created:
+The configuration above tells us some basic information about the pipeline (ID
+and name) and that we want Conduit to start the pipeline automatically. We also
+see a source connector, that uses the `generator` plugin, which is a built-in
+plugin. The connector settings translate into: generate structured data, 1
+record per second. Each generated record should contain an `id` field (type:
+integer) and a `name` field (type: string).
+
+Then we see that we have a destination connector where the data will be written
+to. It uses the `log` plugin, which is a built-in plugin that logs all the
+incoming data. No settings are used here because we're fine with the default
+settings (data will be logged with the `INFO` level).
+ 
+## Run Conduit
+
+With the pipeline configuration being ready, we can run Conduit:
 
 ```shell
-$ ./conduit
-/.../
-2023-04-01T12:34:56+00:00 INF pipeline configs provisioned component=provisioning.Service created=["file-to-file"] deleted=[] pipelines_path=./pipelines
-/.../
+$ conduit --pipelines.path pipeline-generator-to-log.yaml
 ```
 
-Conduit is now running the pipeline `file-to-file` which continuously reads
-lines added to file `./example.in` and copies them to file `./example.out`. Try
-writing a line to `./example.in` and checking the content of `./example.out`.
+Conduit is now running the pipeline. Every second, you should see a line like
+this in the logs:
 
 ```shell
-$ echo "hello conduit" >> example.in
-$ cat example.out
-hello conduit
+2024-10-30T13:31:08+00:00 INF component=plugin connector_id=example-pipeline:example-destination plugin_name=log plugin_type=destination record={"key":"c3RhbGxib2FyZA==","metadata":{"conduit.source.connector.id":"example-pipeline:example-source","opencdc.createdAt":"1730291468447408842","opencdc.payload.schema.subject":"example-pipeline:example-source:payload","opencdc.payload.schema.version":"1"},"operation":"create","payload":{"after":{"id":3312758023475813349,"name":"representativity"},"before":null},"position":"MQ=="}
 ```
 
-![scarf pixel conduit-site-docs-what-is](https://static.scarf.sh/a.png?x-pxid=01346572-0d57-4df3-8399-1425db913a0a)
\ No newline at end of file
+Here's just the generated record:
+
```

Comment:
I'm thinking that it might be easier to see the data with a file destination. We would provide a `tail | jq` command, and a user would be able to see just the record. With the logs, it's a bit hidden.